### ISSUE TITLE
Include Fullscreen Compatibility

### ISSUE THREAD

**Issue by ENTPRESTIGIOUS** [tobsailbot/obs_recording_notification#9]
Was playing a fullscreen but it didn't show up! Other things work like recording my browser! Would be great if it did work while fullscreen since I use OBS to record fullscreen apps most of the time!

**Comment by ZloyKott** [comment]
I agree, support for full-screen applications is needed! And your plugin is the best there is!
